datacommons/examples/statvar.py

```diff
@@ -21,57 +21,78 @@
 
 
 def main():
-    # Dcid for Santa Clara County.
-    sc = 'geoId/06085'
+    data = [
+        {
+            'place': 'geoId/06085',
+            'stat_var': 'Count_Person',
+        },
+        {
+            'place': 'geoId/06085',
+            'stat_var': 'Count_Person',
+            'date': '2018',
+        },
+        {
+            'place': 'geoId/06085',
+            'stat_var': 'Count_Person',
+            'date': '2018',
+            'measurement_method': 'CensusACS5yrSurvey',
+        },
+        {
+            'place': 'geoId/06085',
+            'stat_var': 'UnemploymentRate_Person',
+        },
+        {
+            'place': 'geoId/06085',
+            'stat_var': 'UnemploymentRate_Person',
+            'observation_period': "P1Y",
+        },
+        {
+            'place': 'geoId/06085',
+            'stat_var': 'UnemploymentRate_Person',
+            'observation_period': "P1Y",
+            'measurement_method': "BLSSeasonallyUnadjusted",
+        },
+        {
+            'place':
+                'nuts/HU22',
```

Comment:
no need to newline here and below?

Reply:
This is from yapf formatting, will keep since we plan to enforce google pylint later.